smoke-test/Smoke_Test.postman_collection.json

```diff
@@ -1786,7 +1786,7 @@
 									"raw": ""
 								},
 								"url": {
-									"raw": "{{protocol}}://{{okapiurl}}/circulation/loan-rules/apply-all?item_type_id={{$guid}}&loan_type_id={{$guid}}&patron_type_id={{$guid}}&shelving_location_id={{$guid}}",
+									"raw": "{{protocol}}://{{okapiurl}}/circulation/loan-rules/apply-all?item_type_id={{$guid}}&loan_type_id={{$guid}}&patron_type_id={{$guid}}&location_id={{$guid}}",
```

Comment:
{{$guid}} is a random UUID. This will produce a location lookup failure resulting in a 500 HTTP status code:
https://github.com/folio-org/mod-circulation/blob/370a5b15c0345b8157b23a9d7e47bf7a9312fb7f/src/main/java/org/folio/circulation/resources/AbstractCirculationRulesEngineResource.java#L298
You need to pass in the id of an existing location.